Tests/misc/visitor_test.py

```diff
@@ -0,0 +1,37 @@
+from fontTools.misc.visitor import TTVisitor
+import fontTools.ttLib.tables.otBase as otBase
+import fontTools.ttLib.tables.otTables as otTables
+
+
+class ScalerVisitor(TTVisitor):
+
+    def scale(self, v):
+        return v // 2
+
+@ScalerVisitor.register(otTables.ValueRecord)
+def visit(visitor, obj):
+    attrs = ['XAdvance', 'YAdvance', 'XPlacement', 'YPlacement']
+    for attr in attrs:
+        v = getattr(obj, attr, None)
+        if v is not None:
+            v = visitor.scale(v)
+            setattr(obj, attr, v)
+    return False
+
+@ScalerVisitor.register(otTables.Anchor)
+def visit(visitor, obj):
+    attrs = ['XCoordinate', 'YCoordinate']
+    for attr in attrs:
+        v = getattr(obj, attr)
+        v = visitor.scale(v)
+        setattr(obj, attr, v)
+    return False
+
+
+from fontTools.ttLib import TTFont
+import sys
+
+font = TTFont(sys.argv[1])
```

Comment:
the file is called `visitor_test.py` (with a `_test` suffix) so pytest assumes it is a test module, but it's actually a script that takes argv, hence pytest run fails because as it tries to import the test module. Wrap that into a `if __name__ == "__main__"` or something if you'd like to keep that way

Reply:
I've renamed that to `Snippets/scale-upem.py` now.